components/03-modules/products-grid-item/products-grid-item.js

```diff
@@ -0,0 +1,75 @@
+/* global debounce */
+(function() { // eslint-disable-line
+  'use strict';
+
+  // to calculate image height based on container width
+  const imageRatio = 0.975,
```

Comment:
It's not true... but leave it as it is for now. The product photos on design has changed ratio by adding whitespace around 😞